### DISCUSSION TITLE
How to change Timezone of Action Log

### DISCUSSION THREAD

**Discussion by kamalkeshavani-aiinside** [apache/superset#12934] [Q&A / Help] [answered]
In Superset, by default Action log(Security->Action Log) is recorded in UTC.
Is there a way to change it to a specific time zone?
Is it possible to show this time converted to the browser's local timezone?

**Comment by willbarrett** [comment, marked as answer]
Currently Superset does not support timezone translation and records all actions in UTC.

**Comment by junlincc** [comment]
We have plan to add timezone setting at the system level, as a followup project of Explore Time picker. There will be a SIP for discussion coming soon. @kamalkeshavani-aiinside

**Reply by Kingflyinger** [reply]
when it be able,please let me know
